src/main.ts

```diff
@@ -710,7 +710,14 @@ class Main {
         config;
         if (config.max_games_per_player) {
             const game_count = Object.keys(this.connected_games).filter((game_id) => {
-                return !!this.connected_games[game_id].state?.player_pool[player_id];
+                const state = this.connected_games[game_id]?.state;
+                if (state === undefined) {
+                    return false;
+                }
+                if (state.player_pool !== undefined) {
+                    return !!state.player_pool[player_id];
```

Comment:
```
May 12 16:44:07 ! source-map-support.js:499 TypeError: Cannot read properties of null (reading 'player_pool')
    at /home/siim/projects/gtp2ogs-release/dist/webpack:/gtp2ogs/src/main.ts:717:27
```
means that `state` was null, not undefined
```suggestion
                if (state?.player_pool !== undefined) {
                    return !!state.player_pool[player_id];
```

Reply:
Oh yeah! I had added a `trace.log` for when `state` was `undefined` but I guess it looks like that might be the case in pretty normal circumstances. I'll try to run your suggested form for a moment.